Oracle/OracleJava7JDK.munki.recipe

```diff
@@ -5,43 +5,146 @@
     <key>Description</key>
     <string>Downloads latest release version of Oracle Java 7 JDK and imports into a Munki repository.</string>
     <key>Identifier</key>
-    <string>com.github.jaharmi.munki.OracleJava7JDK</string>
+    <string>com.github.jaharmi.munki.OracleJava7JDK.munki</string>
```

Comment:
That's a copy pasta mistake. Apologies. 

Sent from my iPhone

> On Jul 11, 2016, at 2:56 PM, Clayton Burlison <notifications@github.com> wrote:
> 
> In Oracle/OracleJava7JDK.munki.recipe:
> 
> > @@ -5,43 +5,146 @@
> >      <key>Description</key>
> >      <string>Downloads latest release version of Oracle Java 7 JDK and imports into a Munki repository.</string>
> >      <key>Identifier</key>
> > -    <string>com.github.jaharmi.munki.OracleJava7JDK</string>
> > +    <string>com.github.jaharmi.munki.OracleJava7JDK.munki</string>
> I don't believe this should add .munki to the end of this Identifier. That will break override recipes.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>